.travis.yml

```diff
@@ -5,7 +5,6 @@ cache: pip
 
 python:
   - "3.5"
```

Comment:
can you add 3.6, 3.7 and drop 3.5?

remove also the `if [[ $TRAVIS_PYTHON_VERSION == 3.4 ]];` below and let that command run always